pkgs/development/python-modules/pycflow2dot/default.nix

```diff
@@ -0,0 +1,50 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, cflow
+, graphviz
+, pydot
+, networkx
+, which
+
+, nose
```

Comment:
```suggestion
, which
, nose
```

Reply:
this separation is an intentional grouping
particularly because i'm not sure nose is needed, it is a semantically separate input
is this a significant deviation from accepted style?